### PR TITLE
Add new workflow `test webots build` label

### PR DESCRIPTION
**Pull request by stefaniapedrazzi**
Add new label to be used by the GitHub Action workflow to compile Webots without generating the packages.

This could maybe speed up the tests.
What do you think? Could this be useful?

If yes, I can complete the implementation for all the workflow files.

### CODE REVIEW

**Comment by omichel** [comment]
Yes, that would be useful.
Meanwhile, we should also have a wiki page that explains the actions fired by the tags.

**Comment by stefaniapedrazzi** [comment]
Here is a new page describing the labels:
https://github.com/cyberbotics/webots/wiki/CI-Test-Labels
Also linked from the Git Worklflow page (https://github.com/cyberbotics/webots/wiki/Git-workflow)

I will add it to the wiki home page when this PR is accepted.

**Comment by stefaniapedrazzi** [comment]
`make webots_target -j4` instruction works fine on linux and windows in this PR but not on macOS.
I would need to test on the macOS machine to better debug the issue.

File: .github/workflows/test_suite_linux_develop.yml [hunk 1]
```diff
@@ -12,6 +12,25 @@ defaults:
...
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04, ubuntu-22.04]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v3
+      with:
+         submodules: true
+         fetch-depth: 15
```
Comment (BenjaminDeleze) [inline .github/workflows/test_suite_linux_develop.yml:26]:
```suggestion
      with:
         submodules: true
         fetch-depth: 15
         ref: develop
```

File: .github/workflows/test_suite_mac_develop.yml [hunk 2]
```diff
@@ -4,11 +4,27 @@ on:
...
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v3
+      with:
+         submodules: true
+         fetch-depth: 15
+         ref: develop
+    - name: Build Webots
+      run: |
+        export PATH=/usr/local/bin/:$PATH
```
Comment (BenjaminDeleze) [inline .github/workflows/test_suite_mac_develop.yml:26]: Why do we need to add this to the PATH only for `develop` ?

**Review by BenjaminDeleze — CHANGES_REQUESTED** [review]
Seems good, only some small remarks

**Pull request merged**